hera_cal/noise.py

```diff
@@ -92,7 +98,11 @@ def predict_noise_variance_from_autos(bl, data, dt=None, df=None, nsamples=None)
         df = np.median(np.ediff1d(data.freqs))
 
     ap1, ap2 = split_pol(bl[2])
-    auto_bl1, auto_bl2 = (bl[0], bl[0], join_pol(ap1, ap1)), (bl[1], bl[1], join_pol(ap2, ap2))
+    if auto_ant is None:
+        auto_bl1, auto_bl2 = (bl[0], bl[0], join_pol(ap1, ap1)), (bl[1], bl[1], join_pol(ap2, ap2))
+    else:
+        auto_bl1, auto_bl2 = (auto_ant, auto_ant, join_pol(ap1, ap1)), (auto_ant, auto_ant, join_pol(ap2, ap2))
+    
```

Comment:
Can you add a little unit test for this new functionality?

Reply:
This new functionality is checked in `test_prep_var_for_frop` -- do you have a different way you'd like to check it? The only difference in cutting out the `prep_var_for_frop` wrapper is whether nonfinite values get resassigned.

Reply:
I know it's sort of funky design to test something in `noise` in `test_frf`

Reply:
meh, I think that's good enough